core/src/test/java/org/infinispan/scattered/ScatteredSyncFuncTest.java

```diff
@@ -19,4 +24,37 @@ public ScatteredSyncFuncTest() {
    protected void assertOwnershipAndNonOwnership(Object key, boolean allowL1) {
       Utils.assertOwnershipAndNonOwnership(caches, key);
    }
+
+   public void testCompute() {
```

Comment:
In general, all the test I made before concerned more scenarios.

1) compute/computeIfPresent/computeIfAbsent with existing value in the cache
2) compute/computeIfPresent for an existing value that should be removed if the lambda computes null. computeIfAbsent should do nothing
3) compute/computeIfPresent/computeIfAbsent when a value is not present (computeIfPresent) should do nothing

There are 2 commands Compute for compute/computeIfPresent and ComputeIfAbsent command. You may wanna cover all the scenarios